generators/server/templates/maven/pom.xml

```diff
@@ -92,6 +95,13 @@
             <version>${spring-cloud-aws-version}</version>
         </dependency>
         <%_ } _%>
+        <%_ if (features.includes('elk')) { _%>
+        <dependency>
```

Comment:
In addition to adding this dependency we may also need to add configuration in spring-logback.xml?

Reply:
> In addition to adding this dependency we may also need to add configuration in spring-logback.xml?

Added configuration in application.properties and spring-logback.xml